### ISSUE TITLE
How should different versions of packages be handled?

### ISSUE THREAD

**Issue by pastudan** [ThanksApp/donate-spec#3]

**Comment by erulabs** [comment]
Yeah seems simpler to start with “always assume latest”, but we should design our system so that version is a field we can use the future. Ideally I’d say “search forward” - ie: if no donate field exists with the version we’re using, search future versions for donation information - use newest if current doesn’t have info, use current if it does. Hopefully that explaination makes sense.

If we have a DB with package name -> version -> package.json, then it should be a relatively easy query to find “closest matching donation information”. For sure +1 to starting with only current tho

**Comment by erulabs** [comment]
As an extra idea - we could email maintainers when their donation information changes as a heads up / takeover monitoring mechanism
